include/fastdds/dds/publisher/Publisher.hpp

```diff
@@ -0,0 +1,265 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file Publisher.hpp
+ *
+ */
+
+#ifndef _FASTDDS_PUBLISHER_H_
+#define _FASTDDS_PUBLISHER_H_
+
+#include <fastrtps/fastrtps_dll.h>
+#include <fastdds/rtps/common/Time_t.h>
+#include <fastrtps/attributes/PublisherAttributes.h>
+
+namespace eprosima {
+namespace fastrtps {
+
+namespace rtps
+{
```

Comment:
The coding style specifies that the namespace opening braces must be in the same line as the declaration

Reply:
Removed